src/test/java/cms/gov/madie/measure/services/GroupServiceTest.java

```diff
@@ -743,6 +743,19 @@ public void testUpdateGroupWithNoFunctions() {
     assertNotNull(group);
   }
 
+  @Test
+  public void testUpdateGrroupWithBooleanAsOperandTypeSpecifierName() {
```

Comment:
An extra `r` in Group within the method name.

Reply:
Thanks Joe, I just fixed and pushed the changes